public/styles/component.scss

```diff
@@ -119,3 +119,26 @@ kbn-dis doc-table .kbnDocViewer__warning {
 .header__breadcrumbsWithExtensionContainer .header__breadcrumbsAppendExtension {
     flex-grow: 0;
 }
+
+.osButtonsStyle {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-gap: 10px;
+}
+
+.osButtonsStyleMac {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-gap: 10px;
+}
+
+.message {
+    margin-top: 10px;
+    display: flex;
+    flex-direction: row;
+}
+
+.warningText {
+    color: #936400;
+    font-size: 1rem;
+}
```

Comment:
**question**: where is this class used? Is related to the plugin or this is replacing some properties of the platform?

Reply:
In disuse, removed